cns/NetworkContainerContract.go

```diff
@@ -70,6 +70,12 @@ const (
 	Vxlan = "Vxlan"
 )
 
+// NIC Types
+const (
```

Comment:
maybe make this an enum?
```
type NICType string 
// NIC Types 
const ( 
    NICTypeInfra NICType = "Infra" 
    NICTypeSecondary NICType = "Secondary"
 )
```

Reply:
I like this instead of repeating NICType we can have something like this:
```
type NICType string 
// NIC Types 
const ( 
  Infra NICType = "Infra"  
  Secondary NICType = "Secondary" 
)
```